### PR TITLE
[hi] Add content/hi/docs/reference/glossary/index.md

### PR DESCRIPTION
**Pull request by Polokghosh53**
Hello!

 Localizing [docs/reference/index.md](https://github.com/kubernetes/website/blob/main/content/en/docs/reference/glossary/index.md) to Hindi with respect to #30591 .

Checked out some merged PRs where heading were not localized to Hindi and some keywords like **true** were set the same.

If any query, please do ping me, as this my first PR in Kubernetes/website.

### CODE REVIEW

**Comment by Polokghosh53** [comment]
@anubha-v-ardhan please do check this. Also some technical words like chenopis did not have any Hindi localization.
Please refer: #30591 for the issue.
Requesting reviewers to check, to update any queries possible.

File: content/hi/docs/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+approvers:
+- chenopis
+- abiogenesis-now
```
Comment (anubha-v-ardhan) [inline content/hi/docs/index.md:4]:
```suggestion
```

File: content/hi/docs/index.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+title: शब्दकोष
+layout: शब्दकोष
```
Comment (anubha-v-ardhan) [inline content/hi/docs/index.md:6]:
```suggestion
layout: glossary
```

File: content/hi/docs/index.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+noedit: true
+default_active_tag: मौलिक
```
Comment (anubha-v-ardhan) [inline content/hi/docs/index.md:8]:
```suggestion
default_active_tag: fundamental
```

File: content/hi/docs/index.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+weight: 5
+card:
+  name: संदर्भ
```
Comment (anubha-v-ardhan) [inline content/hi/docs/index.md:11]:
```suggestion
  name: reference
```

File: content/hi/docs/index.md (hunk 1 above)
Comment (anubha-v-ardhan) [reply content/hi/docs/index.md:4]: We are not sure if they are approvers for Hindi version of this page too. We can safely omit `approvers:` here.

File: content/hi/docs/index.md (hunk 2 above)
Comment (anubha-v-ardhan) [reply content/hi/docs/index.md:6]: Should not be translated

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Thank you

**Comment by Polokghosh53** [comment]
@anubha-v-ardhan I was checking out a lot of PRs, it was a bit confusing that which one should I keep as it is. I want to know if this is fine now?

File: content/hi/docs/index.md (hunk 1 above)
Comment (Polokghosh53) [reply content/hi/docs/index.md:4]: So approvers are left out now, right?

Comment (anubha-v-ardhan) [reply content/hi/docs/index.md:4]: Yep its not necessary to have `approvers:` on every page in localization. With Hindi localization, we are not having approvers for pages, so we can omit this section like we did on other pages

**Comment by Polokghosh53** [comment]
@anubha-v-ardhan have updated to the changes you mentioned, please check.

**Review by Polokghosh53 — COMMENTED** [review]
Changes made to the PR according to the suggestions.

**Review by anubha-v-ardhan — COMMENTED** [review]
Oops! The file location seems wrong. This page should be `content/hi/docs/reference/glossary/index.md`

**Comment by Polokghosh53** [comment]
> Oops! The file location seems wrong. This page should be `content/hi/docs/reference/glossary/index.md` not `content/hi/docs/index.md` Can you fix it

I don't know how that happened but I will make sure to change that. I literally memorized the path LOL. Let me see.

**Review by Polokghosh53 — COMMENTED** [review]
I hope this is fine now.

**Comment by anubha-v-ardhan** [comment]
/assign
/lgtm
/approve

Thank you @Polokghosh53

**Comment by Polokghosh53** [comment]
> /assign /lgtm /approve
> 
> Thank you @Polokghosh53

Bigger thanks to you @anubha-v-ardhan .🤗

**Pull request merged**